openmc/tallies.py

```diff
@@ -1468,32 +1528,20 @@ def _outer_product(self, other, binary_op):
             new_name = '({0} {1} {2})'.format(self.name, binary_op, other.name)
             new_tally.name = new_name
 
+        # Query the mean and std dev so the tally data is read in from file
+        # if it has not already been read in.
+        self.mean, self.std_dev, other.mean, other.std_dev
```

Comment:
I'm not sure I understand what's wrong here relative to PEP8, though it's definitely weird for me to see a statement that looks like it doesn't do anything (the comment helps in that regard).